src/AutoPlay.ts

```diff
@@ -89,30 +94,33 @@ class AutoPlay implements Plugin {
     this.flicking = null;
   }
 
-  private play(flicking: Flicking) {
-    this.onStop();
+  public play = () => {
+    const flicking = this.flicking;
+    if (!flicking) return;
+
+    this.stop();
+
+    if (this.mouseEntered || flicking.isPlaying()) return;
 
     this.timerId = window.setTimeout(() => {
       flicking[this.direction === "NEXT" ? "next" : "prev"]();
```

Comment:
nit)
It has nothing to do with this issue, but it would be okay to use enum or union type.